Index: ganttproject/src/net/sourceforge/ganttproject/gui/GanttTaskPropertiesBean.java
===================================================================
diff --git a/ganttproject/src/net/sourceforge/ganttproject/gui/GanttTaskPropertiesBean.java b/ganttproject/src/net/sourceforge/ganttproject/gui/GanttTaskPropertiesBean.java
--- a/ganttproject/src/net/sourceforge/ganttproject/gui/GanttTaskPropertiesBean.java	
+++ b/ganttproject/src/net/sourceforge/ganttproject/gui/GanttTaskPropertiesBean.java	
@@ -245,11 +245,8 @@
     builder.setUiFacade(myUIfacade);
     JPanel colorBox = new JPanel(new BorderLayout(5, 0));
     colorBox.add(builder.createColorComponent(myTaskColorOption).getJComponent(), BorderLayout.WEST);
-    //colorBox.add(Box.createHorizontalStrut(5));
+
     colorBox.add(new JXHyperlink(mySetDefaultColorAction), BorderLayout.CENTER);
-    //colorBox.add(Box.createHorizontalGlue());
-    //colorBox.add(Box.createHorizontalGlue());
-    //colorBox.add(Box.createHorizontalGlue());
 
     propertiesPanel.add(new JLabel(language.getText("colors")));
     propertiesPanel.add(colorBox);
@@ -293,15 +290,6 @@
     propertiesPanel.add(new JLabel(language.getText("file")));
     propertiesPanel.add(fontBox);
 
-    /* TOMAS
-    propertiesPanel.add(new JLabel(language.getText("addTodo")));
-    nameFieldTodo = new JTextField(20);
-    nameFieldTodo.setName("name_of_Todo");
-    propertiesPanel.add(nameFieldTodo);
-    String str = nameFieldTodo.getText();
-    if(str!=null)
-      todoList.add(str);
-    /* --------------------------------*/
     Box todoBox = Box.createHorizontalBox();
     nameFieldTodo = new JTextField(20);
     todoBox.add(nameFieldTodo);
@@ -489,9 +477,7 @@
   /** Apply the modified properties to the selected Tasks */
   public void applySettings() {
     for (int i = 0; i < selectedTasks.length; i++) {
-      // TODO The originalXXX values should not be used,
-      // but the original values should be read from each processed task to
-      // determine whether the value has been changed
+
       TaskMutator mutator = selectedTasks[i].createMutator();
       if (originalName == null || !originalName.equals(getTaskName())) {
         mutator.setName(getTaskName());
